.github/workflows/update-manifest.yaml

```diff
@@ -0,0 +1,55 @@
+name: Repo update microfrontend-manifest
+
+on:
+  repository_dispatch:
+    types: "update_microfrontend_manifest"
+
+jobs:
+  update-dev-manifest:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.client_payload.cluster}} == 'dev-gcp'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - id: 'auth'
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.GCS_SA_KEY_DEV }}'
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+      - name: 'Use gcloud CLI'
+        run: 'gcloud info'
```

Comment:
Er det her debug, eller er det en form for initialisering?

Reply:
Usikker, men vil helst ha den der uansett.